docs/programming_guide/experiment_tracking/experiment_tracking_apis.rst

```diff
@@ -0,0 +1,174 @@
+    "args": {
+      tracking_uri = "file:///{WORKSPACE}/{JOB_ID}/mlruns"
+      "kwargs": {
+        "experiment_name": "hello-pt-experiment",
+        "run_name": "hello-pt-with-mlflow",
+        "experiment_tags": {
+          "mlflow.note.content": "markdown for the experiment"
+        },
+        "run_tags": {
+          "mlflow.note.content": "markdown describes details of experiment"
+        }
+      },
+      "artifact_location": "artifacts"
+    }
+  }
+
+Notice the args{} are user defined, such as tracking_uri, experiment_name, tags etc., and will be specific to which receiver is configured.
+
+The MLflow tracking URL argument ``tracking_uri`` is None by default, which uses the MLflow default URL, ``http://localhost:5000``.
+To make this accessible from another machine, make sure to change it to the correct URL, or point to to the ``mlruns`` directory in the workspace.
+
+::
+
+  tracking_uri = <the Mlflow Server endpoint URL>
+
+::
+
+  tracking_uri = "file:///{WORKSPACE}/{JOB_ID}/mlruns"
+
+You can change other arguments such as experiments, run_name, tags (using Markdown syntax), and artifact location.
+
+Start the MLflow server with one of the following commands:
+
+::
+
+  mlflow server --host 127.0.0.1 --port 5000
+
+::
+
+  mlflow ui -port 5000
+
+For more information with an example walkthrough, see the :github_nvflare_link:`FedAvg with SAG with MLflow tutorial <examples/hello-world/step-by-step/cifar10/sag_mlflow/sag_mlflow.ipynb>`.
+
+
+Streaming metrics to FL clients
+===============================
+
+If streaming metrics to the FL server isn't preferred due to privacy or other concerns, users can alternatively stream metrics to the FL client.
+In such cases, there's no need to add the ``ConvertToFedEvent`` component on the client side.
+Additionally, since we're not streaming to the server side, there's no requirement to configure receivers in the server configuration.
+
+Instead to receive records on the client side, configure the metrics receiver in the client configuration instead of the server configuration.
+
+For example, in the TensorBoard configuration, add this component to ``config_fed_client.conf``:
+
+.. code-block:: yaml
+
+  {
+    "id": "tb_analytics_receiver",
+    "name": "TBAnalyticsReceiver",
+    "args": {"events": ["analytix_log_stats"]}
+  }
+
+Note that the ``events`` argument is ``analytix_log_stats``, not ``fed.analytix_log_stats``, indicating that this is a local event.
+Similarly, you must modify the ``MetricRelay`` component event_type value from ``fed.analytix_log_stats`` to ``analytix_log_stats``.
+
+.. code-block:: yaml
+
+  {
+    id = "metric_relay"
+    path = "nvflare.app_common.widgets.metric_relay.MetricRelay"
+    args {
+      pipe_id = "metrics_pipe"
+      event_type = "analytix_log_stats"
+      # how fast should it read from the peer
+      read_interval = 0.1
+    }
+  },
+
+Then, the metrics will stream to the client.
```

Comment:
Note that, our MetricRelay component is to get things from the API side and fire an event.

By default it is firing a federated event https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/widgets/metric_relay.py#L38

So if we want this to be local event, we change the arg fed_event to False